samples/crypto/test_sample/README.rst

```diff
@@ -0,0 +1,258 @@
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | EdDSA       | `RFC 8032 - Edwards-Curve Digital Signature Algorithm (EdDSA)`_            | `RFC 8032 - Edwards-Curve Digital Signature Algorithm (EdDSA)`_            |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| Hash               | SHA256      | `NIST - SHA`_                                                              | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | SHA512      |                                                                            | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HMAC               | HMAC SHA256 | `NIST - HMAC`_                                                             | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HMAC SHA512 |                                                                            | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HKDF               | HKDF SHA256 | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HKDF SHA512 |                                                                            | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
```

Comment:
EC-JPAKE was added in nrf_security, but has no row(s) in this matrix. We will give input on the test material and the link to the relevant specification

Reply:
See this file:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/tests/crypto/test_cases/test_vectors_ecjpake.c

Shows the test vector source:
https://github.com/ARMmbed/mbed-crypto/blob/master/library/ecjpake.c

I suppose this is the spec source:
https://tools.ietf.org/html/rfc8236#section-3

Reply:
Note: The test-vectors (since they are coming from mbed TLS) can be classified as "custom"

Reply:
curve mode: secp256r1

Reply:
done